app/routers/event.py

```diff
@@ -631,3 +643,17 @@ async def delete_comment(
     cmt.delete_comment(db, comment_id)
     path = router.url_path_for("view_comments", event_id=str(event_id))
     return RedirectResponse(path, status_code=303)
+
+
+@router.get("/check_country_timezone/{country_name}", include_in_schema=False)
+async def check_timezone(
+    country_name,
+    request: Request,
+    db_session: Session = Depends(get_db),
+) -> Response:
+    country_timezone = (
+        db_session.query(Country.timezone)
+        .filter_by(name=country_name)
```

Comment:
I the country_name is wrong, this code will raise an exception.
Do you validate this parameter somewhere?

Reply:
No, but the country name is selected out of a Datalist, which presents all the country names from the DB.. so there is not supposed to be any room for mistakes :see_no_evil:

Reply:
The user can just go to the URL and crash the app :( Please add try/except